Rubberduck.Parsing/VBA/AttributeParser.cs

```diff
@@ -37,7 +37,7 @@ public AttributeParser(IModuleExporter exporter, Func<IVBAPreprocessor> preproce
             cancellationToken.ThrowIfCancellationRequested();
             var component = _projectsProvider.Component(module);
 
-            var path = component.VBE.Kind == VBEKind.Embedded
+            var path = component.VBE.Kind == VBEKind.Hosted
```

Comment:
This leaks a VBE instance.

Reply:
Gah, sorry. I will get the hang of not chaining wrapper calls one day...